__snapshots__/cli.js

```diff
@@ -42,7 +42,7 @@ Options:
    [choices: "dart", "dotnet-yoshi", "elixir", "expo", "go", "go-yoshi", "helm",
                   "java", "java-backport", "java-bom", "java-lts", "java-yoshi",
        "java-yoshi-mono-repo", "krm-blueprint", "maven", "node", "ocaml", "php",
-    "php-yoshi", "python", "ruby", "ruby-yoshi", "rust", "salesforce", "simple",
+    "php-yoshi", "python", "ruby", "ruby-yoshi", "rust", "sfdx", "simple",
```

Comment:
Please leave both `salesforce` and `sfdx` as options for the CLI in case those are being used